examples/notebooks/keras-classification/a - Secure Classification with TFE Keras - Public Training - with DP.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>is this stuff relevant when running in a notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tf-encrypted/tf-encrypted/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>good point, I can remove the flag approach</p>